README.md

```diff
@@ -980,12 +983,32 @@ CredentialAccessBoundary credentialAccessBoundary =
         CredentialAccessBoundary.newBuilder().addRule(rule).build();
 ```
 
+#### Common Usage Pattern
+
 The common pattern of usage is to have a token broker with elevated access generate these downscoped
 credentials from higher access source credentials and pass the downscoped short-lived access tokens
 to a token consumer via some secure authenticated channel for limited access to Google Cloud Storage
 resources.
 
-Using the CredentialAccessBoundary created above in the Token Broker:
+#### Generating Downscoped Tokens
+
+There are two ways to generate downscoped tokens using a
+CredentialAccessBoundary:
+
+* Server-side: Uses the `DownscopedCredentials` class. Each time a
+  downscoped token is needed, the client makes a call to the Security Token Service (STS).
+  This is suitable for applications that require downscoped tokens infrequently.
```

Comment:
Nit: Server side CAB would also be suitable for customers that use a single credential frequently, its more about the number of uniquely downscoped tokens used rather than the frequency of use.
```suggestion
  This is suitable for applications that require downscoped tokens infrequently, or applications that reuse a single downscoped credential many times.
```